GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -48,6 +48,7 @@ public interface IGitTfsRemote
         string Prefix { get; }
         bool ExportMetadatas { get; set; }
         Dictionary<string, string> ExportWorkitemsMapping { get; set; }
+        int? InitialChangeset { get; set; }
```

Comment:
Based on comments I made above for this property, I think this is better off being a method on the interface. Please change this to a method pair: `GetInitialChangeset()` and `SetInitialChangeset(int?)`. This way, you can document under what conditions these proposed methods will throw a `DerivedRemoteException` exception.